uuid-registry.txt

```diff
@@ -151,6 +151,7 @@ e61bb28d-149a-4c1f-b70946823ef5f5ae src4
 33441051-44cd-466a-83a3178478708aea src_lite
 eb0bd14b-7d5e-4dfa-bbe27762adb279f0 swaudiodai
 37c196ae-3532-4282-8a78dd9d50cc7123 testbench
+08aeb4ff-7f68-4c71-86c3c842b4262898 tester
 dd511749-d9fa-455c-b3a713585693f1af tdfb
```

Comment:
oh... ok, this isn't your fault, that's a mistake in the original version - 'd' of course should be before 'e'. So, this isn't a blocker for your PR, but if you make another update, maybe you could fix this too. Otherwise we'll fix it later some time.